pkg/virt-api/rest/kubeproxy.go

```diff
@@ -151,31 +151,9 @@ func GenericResourceProxy(ws *restful.WebService, ctx context.Context, gvr schem
 			Returns(http.StatusNotFound, "Not Found", nil),
 	)
 
-	// TODO, implement watch. For now it is here to provide swagger doc only
-	ws.Route(addWatchGetListParams(
-		ws.GET("/watch/"+gvr.Resource).
-			Produces(mime.MIME_JSON).
-			Operation("watch"+objKind+"ListForAllNamespaces").
-			To(NotImplementedYet).Writes(listExample).
-			Doc("Watch a "+objKind+"List object.").
-			Returns(http.StatusOK, "OK", listExample).
-			Returns(http.StatusNotFound, "Not Found", nil), ws,
-	))
-
-	// TODO, implement watch. For now it is here to provide swagger doc only
-	ws.Route(addWatchNamespacedGetListParams(
-		ws.GET("/watch"+ResourceBasePath(gvr)).
-			Operation("watchNamespaced"+objKind).
-			Produces(mime.MIME_JSON).
-			To(NotImplementedYet).Writes(objExample).
-			Doc("Watch a "+objKind+" object.").
-			Returns(http.StatusOK, "OK", objExample).
-			Returns(http.StatusNotFound, "Not Found", nil), ws,
-	))
-
 	ws.Route(addGetNamespacedListParams(
 		ws.GET(ResourceBasePath(gvr)).
-			Produces(mime.MIME_JSON, mime.MIME_YAML).
+			Produces(mime.MIME_JSON, mime.MIME_YAML, mime.MIME_JSON_STREAM).
```

Comment:
I need to add `mime.MIME_JSON_STREAM` and `watch` parameter to AllNamespaced list as well.

Reply:
Done